### PR TITLE
Banner fixes

### PR DESCRIPTION
**Pull request by jamieliu386**
Fixes: #46 

Makes the countdown's font smaller on `xs` screens. Also fixes the text alignment for "Hack on the Hill 7" and fixes the date used for the countdown.

image

### CODE REVIEW

File: src/components/HomePage/Banner.js [hunk 1]
```diff
@@ -143,13 +149,15 @@ function Banner() {
 		completed: PropTypes.bool.isRequired
 	};
 
+	const smallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+
 	return (
 		<div className={classes.container}>
```
Comment (TimothyGu) [inline src/components/HomePage/Banner.js:155]: Idea: reduce the padding on the `.container` for smaller screen sizes. I think `padding: '12px 4px'` looks good.

File: src/components/HomePage/Banner.js [hunk 2]
```diff
@@ -40,7 +42,11 @@ const useStyles = makeStyles(theme => ({
 	timer: {
 		color: 'white',
 		fontWeight: 600,
-		fontSize: '4em'
+		fontSize: '4em',
+		textAlign: 'center',
+		[theme.breakpoints.down('xs')]: {
```
Comment (TimothyGu) [inline src/components/HomePage/Banner.js:47]: I think the addition of the rule

```js
[theme.breakpoints.down('sm')]: {
  fontSize: '3em'
}
```

would work even better than this change alone. Right now at the `sm` size, the timer still appears bigger than the "Hack on the Hill 7" text itself, which looks a bit strange.

File: src/components/HomePage/Banner.js [hunk 3]
```diff
@@ -40,7 +42,11 @@ const useStyles = makeStyles(theme => ({
 	timer: {
 		color: 'white',
 		fontWeight: 600,
-		fontSize: '4em'
+		fontSize: '4em',
+		textAlign: 'center',
```
Comment (TimothyGu) [inline src/components/HomePage/Banner.js:46]: The `textAlign` change doesn't appear necessary here, and I suspect that the `textAlign` change above might not be necessary either. I think the `alignItems` fix below might have rendered these unnecessary.

**Comment by jamieliu386** [comment]
> While it does admit extra space between the colon and the number on small screens, I think it superior to making the units not readable at all.

How about making the font size bigger but also changing the units for small screens? i.e. dd : hh : mm : ss or something similar? I really don't like the look of the extra space between the number and colon.

File: src/components/HomePage/Banner.js [hunk 4]
```diff
@@ -103,7 +106,7 @@ function Banner() {
 					>
 						{days.toString().padStart(2, '0')}
 					</Box>
-					:
+					<div style={{ padding: '0px 5px' }}>:</div>
```
Comment (TimothyGu) [inline src/components/HomePage/Banner.js:108]: Can you move these styles into a class, like `className={classes.colon}`?

Comment (TimothyGu) [reply src/components/HomePage/Banner.js]: I've taken care of it.

**Comment by jamieliu386** [comment]
Thanks for the fixes, looks good!

**Pull request merged**